extensions-api/src/main/scala/pl/touk/nussknacker/engine/api/deployment/ScenarioActivity.scala

```diff
@@ -103,6 +99,25 @@ sealed trait ScenarioActivity {
   def scenarioVersionId: Option[ScenarioVersionId]
 }
 
+sealed trait DeploymentRelatedActivity extends ScenarioActivity {
+  def result: DeploymentRelatedActivityResult
+  def dateFinished: Option[Instant]
+}
+
+sealed trait BatchDeploymentRelatedActivity extends DeploymentRelatedActivity
+
+sealed trait DeploymentRelatedActivityResult
```

Comment:
The ScenarioActivity does not have state, but deployment-related activities have final result - success/failure.

Reply:
can we rename it to `DeploymentResult`? because for me it's a result of the operation (here, the operation is deployment), not an activity result.